packages/material-ui/src/OutlinedInput/NotchedOutline.js

```diff
@@ -32,7 +32,7 @@ export const styles = theme => {
     },
     /* Styles applied to the legend element. */
     legend: {
-      textAlign: align,
+      textAlign: 'left',
       padding: 0,
```

Comment:
This is now always left even if `theme.direction` is `rtl`. Not sure what this change has to do with the margin.

Reply:
It's the fix @olivertasinari suggested for #12914.

If it's not correct, I'll look into it tomorrow

Reply:
I thought `align` became obsolete but it is still used for transition.

Reading the issue and looking at https://deploy-preview-12939--material-ui.netlify.com/demos/text-fields/#outlined this did indeed do the trick.

Everything good :+1: